lib/internal/Magento/Framework/Data/Collection.php

```diff
@@ -492,7 +492,7 @@ public function clear()
      *
      * Returns array with results of callback for each item
      *
-     * @param string $callback
+     * @param string|callable $callback
```

Comment:
It can even be just `callable` (without `string`).

Reply:
@oliverklee true :) https://github.com/magento/magento2/pull/13373#issuecomment-361098279

Reply:
Done & squashed into a single commit. Please let me know if anything else is needed.